### ISSUE TITLE
Waves destroy

### ISSUE THREAD

**Issue by sergeyzwezdin** [fians/Waves#159]
While there are `Waves.init()` and `Waves.attach()` exists, there is no `Waves.destroy()` and `Waves.detach()`.

At the same time there is set of `addEventListener()` calls, so Waves subscribes on control events some time, but never unsubscribes on this.
For example: https://github.com/fians/Waves/blob/master/src/js/waves.js#L442-L462

So it's quite possible to get memory leaks on large application because of using Waves. Could you please add `destroy()` and `detach()` methods the Waves API, or point how to workaround this issue?

**Comment by fians** [comment]
Hi @sergeyzwezdin, nice idea. I will work on this. Thanks!

**Comment by fians** [comment]
Added to v0.8.0
